### PR TITLE
Enable/disable command

### PR DESCRIPTION
**Pull request by Roosterseatyou**
This allows for the effects to be toggled on a per-level scale, however, it can be disabled in the config for any evil server owners.

### CODE REVIEW

File: src/main/java/ml/treecaptcha/uwuify/spigot/commands/UwUCommands.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+                    }
+                }
+            }
+            return false;
+        }
+        return false;
+    }
+
+    public static boolean isEnabled(Player p, NamespacedKey key) {
+        return !Boolean.parseBoolean(p.getPersistentDataContainer().getOrDefault(Uwuify.UWUIFY_KEY, PersistentDataType.STRING, "true")) && Boolean.parseBoolean(p.getPersistentDataContainer().getOrDefault(key, PersistentDataType.STRING, "true"));
```
Comment (treecaptcha) [inline src/main/java/ml/treecaptcha/uwuify/spigot/commands/UwUCommands.java:55]: is there a reason for it to be a string and not a boolean?

File: src/main/resources/config.yml [hunk 2]
```diff
@@ -35,6 +35,9 @@ player-names-uwuify: false
 # this settings is very dangerous
 books-uwuify: false
 
+# this setting is for the uwuify commands
+allow-toggle: true
```
Comment (treecaptcha) [inline src/main/resources/config.yml:39]: could we do a permission for allowing toggle?

File: src/main/java/ml/treecaptcha/uwuify/spigot/commands/UwUCommands.java [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+        p.getPersistentDataContainer().set(key, PersistentDataType.STRING,
+                (Boolean.parseBoolean(p.getPersistentDataContainer().get(key, PersistentDataType.STRING))) ? "false" : "true");
+        p.sendMessage("UwUify " + (Boolean.parseBoolean(p.getPersistentDataContainer().get(key, PersistentDataType.STRING)) ? "disabled" : "enabled"));
+        return true;
+    }
+
+    @Override
+    public @Nullable List<String> onTabComplete(@NotNull CommandSender commandSender, @NotNull Command command, @NotNull String s, @NotNull String[] strings) {
+        if (Uwuify.ALLOW_TOGGLE && commandSender instanceof Player p) {
+            return List.of("chat", "signs", "books", "names", "join", "all", "animal_names");
```
Comment (treecaptcha) [inline src/main/java/ml/treecaptcha/uwuify/spigot/commands/UwUCommands.java:68]: Could this be cached?

**Review by treecaptcha — CHANGES_REQUESTED** [review]
Nice!

File: src/main/java/ml/treecaptcha/uwuify/spigot/commands/UwUCommands.java (hunk 1 above)
Comment (Roosterseatyou) [reply src/main/java/ml/treecaptcha/uwuify/spigot/commands/UwUCommands.java:55]: Besides my own stupidity, no. Will work on creating a Boolean PersistentDataType now :)

File: src/main/resources/config.yml (hunk 2 above)
Comment (Roosterseatyou) [reply src/main/resources/config.yml:39]: Will do 👍🏻

File: src/main/java/ml/treecaptcha/uwuify/spigot/commands/UwUCommands.java (hunk 3 above)
Comment (Roosterseatyou) [reply src/main/java/ml/treecaptcha/uwuify/spigot/commands/UwUCommands.java:68]: Assuming I know what you mean, yes.

File: src/main/java/ml/treecaptcha/uwuify/spigot/commands/UwUCommands.java (hunk 1 above)
Comment (treecaptcha) [reply src/main/java/ml/treecaptcha/uwuify/spigot/commands/UwUCommands.java:55]: @Roosterseatyou I think I saw byte available we could do 0x00 is false and 0x01 is true

Comment (Roosterseatyou) [reply src/main/java/ml/treecaptcha/uwuify/spigot/commands/UwUCommands.java]: 👍

**Review by treecaptcha — APPROVED** [review]

**Pull request merged**